Std/Data/List/Lemmas.lean

```diff
@@ -181,6 +181,9 @@ theorem mem_iff_append {a : α} {l : List α} : a ∈ l ↔ ∃ s t : List α, l
 
 /-! ### map -/
 
+@[simp]
+theorem map_eq_map (f : α → β) (l : List α) : f <$> l = map f l := rfl
```

Comment:
I don't think there is a style guidance about this but if it can all fit in one line then I tend to prefer not breaking a line after `@[simp]`. This applies elsewhere but it's just nitpick.